x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/RBACEngine.java

```diff
@@ -408,7 +408,12 @@ public void authorizeIndexAction(
     }
 
     private static boolean allowsRemoteIndices(TransportRequest transportRequest) {
-        return transportRequest instanceof IndicesRequest.Replaceable replaceable && replaceable.allowsRemoteIndices();
+        /// MP TODO: best way to handle this?
```

Comment:
nit: you may reference this new issue in the comment here: https://github.com/elastic/elasticsearch/issues/105598

Reply:
Done. Thanks for the link.